File: src/main/java/com/aplicativo/trashmap/enums/STATUSCAMINHAO.java
package com.aplicativo.trashmap.enums;

public enum STATUSCAMINHAO {
    ATIVO ("Ativo"),
    INATIVO ("Inativo"),
    EM_MANUTENCAO ("Em Manutenção");

    private final String descricao;

    STATUSCAMINHAO (String descricao){
        this.descricao=descricao;
    }

    public String getDescricao(){
        return descricao;
    }
}

File: src/main/kotlin/com/sistema/trashmap/domain/enum/StatusCaminhao.kt
package com.sistema.trashmap.domain.enum

import com.fasterxml.jackson.annotation.JsonValue

enum class StatusCaminhao(val descricao: String) {

    ATIVO("Ativo"),
    // Caminhão disponível e em operação
    // Pode ser exibido no mapa e receber rotas de coleta

    INATIVO("Inativo"),
    // Caminhão não está em operação
    // Não participa do rastreamento em tempo real

    EM_MANUTENCAO("Em Manutenção");
    // Caminhão em manutenção preventiva ou corretiva
    // Temporariamente indisponível para rotas de coleta


}

